js/src/reports/chart-section.js

```diff
@@ -49,8 +47,7 @@ export default function ChartSection( { metrics, loaded, intervals } ) {
 
 	const chartType = getChartTypeForQuery( query );
 	const valueType = isCurrency ? 'currency' : 'number';
-	const localizedFormatFn = ( number ) =>
-		formatFn( number, storeCurrencyConfig, adsCurrencyConfig );
+	const localizedFormatFn = formatFn.bind( visibleMetric );
```

Comment:
❓ What's exactly the intent of this change?

Reply:
I think now `formtFn` is the new `formatFn` function that only receives 2 arguments (the value and true for use ads format instead of store format as far as I understand).